test/functional/test_runner.py

```diff
@@ -407,6 +409,9 @@ def __repr__(self):
         if self.status == "Passed":
             color = BLUE
             glyph = TICK
+        if self.status == "Passed with warnings":
+            color = ORANGE
```

Comment:
`ORANGE` would need to be a constant defined at the top of this file (see around L47)